### PR TITLE
Fix UI deployment , backport to release-4.12 branch

### PR DESCRIPTION
**Pull request by OdedViner**
#7543
Fix UI deployment

### CODE REVIEW

File: ocs_ci/ocs/ui/base_ui.py [hunk 1]
```diff
@@ -334,9 +333,11 @@ def get_page_hash():
             time.sleep(sleep_time)
             page_hash_new = get_page_hash()
             if retry_counter == retries:
-                raise PageNotLoaded(
+                logger.error(
```
Comment (petr-balogh) [inline ocs_ci/ocs/ui/base_ui.py:336]: Is it intention here to not raise exception if retry_counter == retries and just do logger.error instead?
Will the code continue as I think the exception prevent this before and fail?

Comment (OdedViner) [reply ocs_ci/ocs/ui/base_ui.py:336]: Correct.. if we can't click on a specific button. Selenium will raise an exception.

**Comment by OdedViner** [comment]
PR validation pass https://url.corp.redhat.com/c522186

File: ocs_ci/ocs/ui/helpers_ui.py [hunk 2]
```diff
@@ -121,7 +121,7 @@ def ui_add_capacity_conditions():
     ):
         logger.info(f"Add capacity via UI is not supported on platform {platform}")
         return False
-    elif ocp_version not in ("4.7", "4.8", "4.9", "4.10", "4.11", "4.12"):
+    elif ocp_version not in ("4.7", "4.8", "4.9", "4.10", "4.11", "4.12", "4.13"):
```
Comment (dahorak) [inline ocs_ci/ocs/ui/helpers_ui.py:124]: I'm not sure, if we should keep the `4.13` related changes in the backport for `release-4.12` branch. @petr-balogh, @clacroix12, @vavuthu  WDYT?

File: ocs_ci/ocs/ui/views.py [hunk 3]
```diff
@@ -1157,6 +1157,19 @@
...
+        "generic": generic_locators,
+        "deployment": {
+            **deployment,
+            **deployment_4_7,
+            **deployment_4_9,
+            **deployment_4_10,
+            **deployment_4_11,
+            **deployment_4_12,
+        },
+    },
```
Comment (dahorak) [inline ocs_ci/ocs/ui/views.py:1172]: Same as above, I'm not sure, if we should keep the `4.13` related changes in the backport for `release-4.12` branch.

**Review by dahorak — COMMENTED** [review]
I'm not sure about `4.13` related changes in PR for `release-4.12` branch, otherwise LGTM.

File: ocs_ci/ocs/ui/helpers_ui.py (hunk 2 above)
Comment (OdedViner) [reply ocs_ci/ocs/ui/helpers_ui.py:124]: done

File: ocs_ci/ocs/ui/views.py (hunk 3 above)
Comment (OdedViner) [reply ocs_ci/ocs/ui/views.py:1172]: done

**Review by dahorak — APPROVED** [review]

**Review by petr-balogh — APPROVED** [review]

**Pull request merged**